tmt/config/models/themes.py

```diff
@@ -44,14 +44,14 @@ def get_style(self, field: str) -> Style:
         A safer, type-annotated variant of ``getattr(theme, field)``.
         """
 
-        if field not in self.__fields__:
+        if field not in self.model_fields_set:
             raise tmt.utils.GeneralError(
                 f"No such theme field '{self.__class__.__name__.lower()}.{field}'."
             )
 
-        # attr-defined: `type_` seems to be invisible to mypy. It might
-        # be causes by Pydantic v1/v2 difference.
-        if self.__fields__[field].type_ is not Style:  # type: ignore[attr-defined]
+        # Using model_fields is deprecated and will be removed in pyright v3
```

Comment:
```suggestion
        # Using model_fields is deprecated and will be removed in pydantic v3
```

@LecrisUT I guess this is a typo